src/resources/promoSheet/promoSheetCollectionQuery.ts

```diff
@@ -55,11 +55,11 @@ export async function fetchPromoSheetCollection({ order }: PromoSheetCollectionA
 
 export function usePromoSheetCollectionQuery(
   { order }: PromoSheetCollectionArgs = {},
-  { enabled, refetchInterval = 30_000 }: { enabled?: boolean; refetchInterval?: number } = {}
+  config: QueryConfig<PromoSheetCollectionResult, Error, PromoSheetCollectionQueryKey> = {}
 ) {
   return useQuery(promoSheetCollectionQueryKey({ order }), promoSheetCollectionQueryFunction, {
-    enabled,
     staleTime: defaultStaleTime,
-    refetchInterval,
+    refetchInterval: 60_000,
```

Comment:
Probably doesn't even need to refetch this often right? Would guess in most cases there's a check on launch which is unlikely to change within a minute.

Not sure of the details of `refetchInterval` behavior, but could probably be 5 minutes without making much of a difference in practice.